microk8s-installation.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+sudo snap install microk8s --classic
+sudo microk8s status --wait-ready
+sudo microk8s enable dns hostpath-storage registry
+sudo usermod -a -G microk8s $USER
+sudo chown -f -R $USER ~/.kube
+sudo microk8s kubectl -n kube-system rollout status -w deployment/hostpath-provisioner
+sudo microk8s kubectl -n kube-system rollout status -w deployment/coredns
+sudo microk8s kubectl -n container-registry rollout status -w deployment/registry
+sg microk8s -c "juju bootstrap microk8s k8s-ctrl"
```

Comment:
Wouldnt be necessary to run juju switch k8s-ctrl  before add-model??
I think integration_test.yaml had this command.

Reply:
It is not. When you run juju bootstrap, you will be on that controller by default after the bootstrap ends. You would need to execute a switch if you need to bootstrap a controller and then switch to another one.

Reply:
You might want to put `$USER` in quotes like `"$USER"` to keep shellcheck happy, although not required.